README.md

```diff
@@ -7,7 +7,41 @@ This repository provides KubeVirt-specific Tekton tasks, which focus on:
 - Executing commands in VMs
 - Manipulating disk images with virt-customize [WIP]
 
-## Usage and Deployment
+## Deployment
+
+### On Kubernetes
+
+Run the following command to install KubeVirt Tekton tasks in an active namespace.\
+Note: this needs to be run in every namespace the tasks will be used in (due to rbac privileges). 
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/kubernetes/namespace-rbac/kubevirt-tekton-tasks.yaml
+```
+
+If the pipeline needs to access resources (VMs, PVCs, etc.) across different namespaces (cluster privileges), the following commands need to be run instead.
+
+```bash
+TARGET_NAMESPACE="$(kubectl config view --minify --output 'jsonpath={..namespace}')"
+wget -qO - https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/kubernetes/cluster-rbac/kubevirt-tekton-tasks.yaml | sed "s/TARGET_NAMESPACE/$TARGET_NAMESPACE/" | kubectl apply -f -
+```
+
+### On OpenShift
+
+Run the following command to install KubeVirt Tekton tasks in an active namespace. These also include OpenShift specific tasks.\
```

Comment:
I cannot parse the English of "These also include OpenShift specific tasks". And why's the trailing backslash?

Reply:
I rewrote that part a bit. Originally the reason behind the backslash was that I wanted the note on a new line but not in a new paragraph.